Context/Adapter/JMSContextAdapter.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Context\Adapter;
+
+use FOS\RestBundle\Context\ContextInterface;
+use FOS\RestBundle\Context\GroupableContextInterface;
+use FOS\RestBundle\Context\MaxDepthContextInterface;
+use FOS\RestBundle\Context\SerializeNullContextInterface;
+use FOS\RestBundle\Context\VersionableContextInterface;
+use JMS\Serializer\Context as JMSContext;
+use JMS\Serializer\DeserializationContext as JMSDeserializationContext;
+use JMS\Serializer\SerializationContext as JMSSerializationContext;
+
+/**
+ * @author Ener-Getick <egetick@gmail.com>
+ *
+ * @internal
+ */
+final class JMSContextAdapter
```

Comment:
I would add a ``@TODO`` item for such stuff ... even if we are going to do this change right afterwards